manager/src/main/java/com/redhat/service/bridge/manager/actions/connectors/ConnectorActionUtils.java

```diff
@@ -0,0 +1,38 @@
+package com.redhat.service.bridge.manager.actions.connectors;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.BooleanNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import static com.redhat.service.bridge.manager.actions.connectors.ConnectorAction.PROCESSORS_PARAMETER;
+
+public class ConnectorActionUtils {
+
+    public static final String LOG_PROCESSOR_PARENT_PARAMETER = "log";
+    public static final String LOG_PROCESSOR_MULTILINE_PARAMETER = "multiLine";
+    public static final String LOG_PROCESSOR_SHOWHEADERS_PARAMETER = "showHeaders";
+
+    public static ObjectNode addLogProcessorToDefinition(ObjectMapper mapper, ObjectNode definition) {
```

Comment:
I'd use an `@ApplicationScoped` bean, inject the ObjectMapper and pass only the definition as the parameter

Reply:
Done :+1: